bitrise.yml

```diff
@@ -635,7 +635,7 @@ workflows:
         - pull_request_merge_branch: ""
         - branch_dest: "master"
         - branch: "master"
-        - commit: ""
+        - commit: "commit"
```

Comment:
this seems to be a strange commit hash to me, did we intentionally added this here?

Reply:
This is also needed based on our conversation this morning, that a commit also has to be provided.

Reply:
I see, this case it would be great to specify the valid commit hash of the PR commit